electricitymap/contrib/capacity_parsers/tests/test_REE.py

```diff
@@ -0,0 +1,120 @@
+from datetime import datetime
+
+from electricitymap.contrib.capacity_parsers import REE
+
+
+class FakeResponse:
+    def __init__(self, status_code: int, payload: dict):
+        self.status_code = status_code
+        self._payload = payload
+
+    def json(self):
+        return self._payload
+
+
+class FakeSession:
+    def __init__(self, response: FakeResponse):
+        self._response = response
+        self.last_call = None
+
+    def get(self, url, params=None):
+        self.last_call = (url, params)
+        return self._response
```

Comment:
These should not be faked. There is already mock versions in the request library that should be used for testing.